src/Serialization/main.jl

```diff
@@ -310,6 +334,9 @@ function register_serialization_type(ex::Any, str::String, uses_id::Bool, uses_p
       # Types like ZZ, QQ, and ZZ/nZZ do not require ids since there is no syntactic
       # ambiguities in their encodings.
 
+      # add list of possible attributes to save for a given type to a global dict
+      register_attr_list($ex, $attrs)
```

Comment:
```suggestion
      Oscar.register_attr_list($ex, $attrs)
```
to make it consistent with the rest of this function once https://github.com/oscar-system/Oscar.jl/pull/4021 is merged